### PR TITLE
add documentation for configurable repo serving in api mode

### PR DESCRIPTION
**Pull request by mmianl**

### CODE REVIEW

File: content/doc/configuration.md [hunk 1]
```diff
@@ -138,6 +139,7 @@ Options:
     default value when log level is `default`
 -   `logFormat` specifies the log format to use, can be `default` or `json`, when format is `json`
      all logs have a `time` key with timestamps in RFC3339 format and a `level` key with the log level
+-   `serveInAPIMode` enables serving published repos from public/ subdirectory of aptly’s root while running aptly in the api mode. Published repos will be served from `/repos`.
```
Comment (aol-nnov) [inline content/doc/configuration.md:142]: This is not totally correct. It serves files from a `rootDir` of every configured `FileSystemPublishEndpoints`

Your description applies to "default" mode only, when none of `FileSystemPublishEndpoints` are configured.

Comment (mmianl) [reply content/doc/configuration.md:142]: What description would you suggest?

Comment (aol-nnov) [reply content/doc/configuration.md:142]: maybe, something like this?

`serveInAPIMode` enables serving published repos according to configured `FileSystemPublishEndpoints` while running aptly in the api mode. If there are no `FileSystemPublishEndpoints`, it serves public/ subdirectory of aptly’s root.
(cut here, but if you feel like adding an example, here we go)

i.e. if you have `FileSystemPublishEndpoints` configured as follows:
```json
"FileSystemPublishEndpoints": {
    "bootstrap": {
      "rootDir": "/srv/publishes",
     ... other params
    },
    "some_other": {
      "rootDir": "/opt/another",
     ... other params
    }
```
Then `/srv/publishes` is accessible via `http://aptlyhost/repos/bootstrap/` and `/opt/another` via `http://aptlyhost/repos/some_other/` accordingly.

If `FileSystemPublishEndpoints` are not configured, default publish location is accessible via `http://aptlyhost/repos/-/`

**Comment by mmianl** [comment]
@randombenj I think this should be merged

**Pull request merged**